### PR TITLE
add cross browser support for image-set() css ref in alternative image formats example

### PR DESCRIPTION
**Pull request by digi-booster**
…e formats example

### CODE REVIEW

**Comment by digi-booster** [comment]
@teoli2003 Can you review this?

**Review by rachelandrew — CHANGES_REQUESTED** [review]
You need to repeat any change inside the textarea or it will be overwritten onload. 

These are very easy to test by opening them in a browser.

**Comment by digi-booster** [comment]
> You need to repeat any change inside the textarea or it will be overwritten onload.
> 
> These are very easy to test by opening them in a browser.

Is this correct now?

**Comment by digi-booster** [comment]
Hi @rachelandrew, I waiting for your re-review of this pr.

**Comment by rachelandrew** [comment]
To open a file in a browser? Double click the file icon, it will open in the browser.

File: images/image-set-type.html [hunk 1]
```diff
@@ -30,6 +33,9 @@
 
     <textarea class="playable playable-css" style="height: 150px;">
```
Comment (rachelandrew) [inline images/image-set-type.html:34]: Have you tested that this does not cause a scrollbar? I think height: 150px will not be tall enough for people to see the code.

Comment (digi-booster) [inline images/image-set-type.html:34]:
```suggestion
    <textarea class="playable playable-css" style="height: 155px;">
```
height: 150px;
before

height: 155px;
after

**Review by rachelandrew — APPROVED** [review]

**Pull request merged**